selfdrive/ui/translations/main_th.ts

```diff
@@ -0,0 +1,1327 @@
+<context>
+    <name>PrimeAdWidget</name>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="187"/>
+        <source>Upgrade Now</source>
+        <translation>อัพเกรดเดี๋ยวนี้</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="192"/>
+        <source>Become a comma prime member at connect.comma.ai</source>
+        <translation>สมัครสมาชิก comma prime ได้ที่ connect.comma.ai</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="199"/>
+        <source>PRIME FEATURES:</source>
+        <translation>คุณสมบัติของ PRIME:</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="204"/>
+        <source>Remote access</source>
+        <translation>การเข้าถึงระยะไกล</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="204"/>
+        <source>1 year of storage</source>
+        <translation>จัดเก็บข้อมูลนาน 1 ปี</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="204"/>
+        <source>Developer perks</source>
+        <translation>สิทธิพิเศษสำหรับนักพัฒนา</translation>
+    </message>
+</context>
+<context>
+    <name>PrimeUserWidget</name>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="126"/>
+        <source>✓ SUBSCRIBED</source>
+        <translation>✓ สมัครสำเร็จ</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="132"/>
+        <source>comma prime</source>
+        <translation>comma prime</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="138"/>
+        <source>CONNECT.COMMA.AI</source>
+        <translation>CONNECT.COMMA.AI</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/prime.cc" line="151"/>
+        <source>COMMA POINTS</source>
+        <translation>คะแนน COMMA</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../qt/text.cc" line="36"/>
+        <source>Reboot</source>
+        <translation>รีบูต</translation>
+    </message>
+    <message>
+        <location filename="../qt/text.cc" line="41"/>
+        <source>Exit</source>
+        <translation>ปิด</translation>
+    </message>
+    <message>
+        <location filename="../qt/util.cc" line="21"/>
+        <source>dashcam</source>
+        <translation>กล้องติดรถยนต์</translation>
+    </message>
+    <message>
+        <location filename="../qt/util.cc" line="21"/>
+        <source>openpilot</source>
+        <translation>openpilot</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/util.cc" line="82"/>
+        <source>%n minute(s) ago</source>
+        <translation type="unfinished">
+            <numerusform></numerusform>
```

Comment:
Could you add the few unfinished translations with the rebase?

Reply:
Yes, I will do it.

Reply:
Hello. Any updates on doing these?

Reply:
> 

Sorry, I will try to finish it today.

Reply:
thanks!